modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -64,13 +73,11 @@ For details, see xref:policies-landing-page.adoc[Policies].
 +
 For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
 
-<8> The tracking registration status of each API: *Active*, *Inactive*, *Deploying*, or *Unregistered*.  
-+
-A status of *Unregistered* means that Anypoint Platform has never tracked the endpoint for this API version. Either you have entered a URL for an API or proxy that is not yet deployed, or you have declared an endpoint that is hosted somewhere other than an API Gateway and need to proxy that endpoint so that the platform can track it.
-+
-The endpoint must have a tracking registration status of *Active* for governance policies and SLA tiers to function.
+<9> The percentage of API requests that resulted in errors (in the previous 24 hours). For additional information about API Functional Monitoring, see xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
```

Comment:
It's not directly related to functional monitoring, but to "Monitoring" product.
https://docs.mulesoft.com/monitoring/

Also, if "No Data" is shown, Monitoring is not enabled for such API.
https://docs.mulesoft.com/monitoring/configure-monitoring-cloudhub#enable-automatic-monitoring-for-apps-that-you-plan-to-deploy-to-cloudhub